airflow/www/templates/airflow/graph.html

```diff
@@ -53,7 +53,7 @@
   {% for op in operators %}
   <div class="legend_item"
        style="border-width:1px;float:left;background:{{ op.ui_color }};color:{{ op.ui_fgcolor }};">
-    {{ op.__name__ }}
+    {{ op.name }}
```

Comment:
what is this change about?

Reply:
This is because I don't pass the operators as class anymore, this would require to also parse the DAG file again. Now doing a distinct on all task_instance for a single run from database.